README.md

```diff
@@ -29,4 +29,127 @@ await linodeModule.uploadFileToLinodeBucket(accessKeyId,
 const fileURL = await linodeModule.fetchObjectUrl(accessToken, 
     clusterId, bucketName, objectName);
 ```
-    
+
+## Code Guardian
+[![<app> build verification](https://github.com/aicore/linode-object-storage-lib/actions/workflows/build_verify.yml/badge.svg)](https://github.com/aicore/linode-object-storage-lib/actions/workflows/build_verify.yml)
+
+<a href="https://sonarcloud.io/summary/new_code?id=aicore_template-nodejs-ts">
```

Comment:
Need changing to `id=aicore_linode-object-storage-lib` from `id=aicore_template-nodejs-ts` and similar below

Reply:
Also code guardian section is best laid out on top of readme to get quick statuses of the repo

Reply:
@abose Agreed, will refactor the README based on the comments above.